Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -879,14 +879,16 @@ def addCustomise(self,unsch=0):
 		final_snippet += '\n# End of customisation functions\n'
 
 	### now for a useful command
-	if unsch==1 or not self._options.runUnscheduled:
-		if self._options.customise_commands:
-			import string
-			final_snippet +='\n# Customisation from command line'
-			for com in self._options.customise_commands.split('\\n'):
-				com=string.lstrip(com)
-				self.executeAndRemember(com)
-				final_snippet +='\n'+com
+	return final_snippet
+
+    def addCustomiseCmdLine(self,unsch=0):
```

Comment:
whoops - yes!

On Aug 11, 2016, at 4:23 PM, Kevin Pedro <notifications@github.com<mailto:notifications@github.com>> wrote:


In Configuration/Applications/python/ConfigBuilder.py<https://github.com/cms-sw/cmssw/pull/15426#discussion_r74431385>:

> @@ -879,14 +879,16 @@ def addCustomise(self,unsch=0):
>               final_snippet += '\n# End of customisation functions\n'
>
>       ### now for a useful command
> -     if unsch==1 or not self._options.runUnscheduled:
> -             if self._options.customise_commands:
> -                     import string
> -                     final_snippet +='\n# Customisation from command line'
> -                     for com in self._options.customise_commands.split('\\n'<smb://n'>):
> -                             com=string.lstrip(com)
> -                             self.executeAndRemember(com)
> -                             final_snippet +='\n'+com
> +     return final_snippet
> +
> +    def addCustomiseCmdLine(self,unsch=0):


unsch parameter is no longer needed here

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/cms-sw/cmssw/pull/15426/files/5a7de1e86738a693a7bd0871f8d0b98080c45e8b..396c93c5a325c0f10306eab8f66f2a640c773c00#r74431385>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AEzyw7x-0wqBJuLFm9tC2hPoRfJQUuV8ks5qezBsgaJpZM4JiHCG>.